exercises/anagram/test/anagram_test.dart

```diff
@@ -16,70 +16,80 @@ void main() {
 }
 
 void simpleTests() {
-  test("no matches", () {
-    final List<String> result = anagram.findAnagrams("diaper", ["hello", "world", "zombies", "pants"]);
+  test('no matches', () {
+    final List<String> result = anagram.findAnagrams('diaper', <String>['hello', 'world', 'zombies', 'pants']);
```

Comment:
`<String>` type annotation on the list literal is not required. `findAnagrams` only accepts List<String> as it's second argument. Was this added by the create_exercise tool?

Reply:
Same for rest of the list literals in this file.